django-stubs/db/models/expressions.pyi

```diff
@@ -121,15 +132,13 @@ class F(Combinable):
     def asc(
```

Comment:
Do we still need these method? They are duplicating the `BaseExpression` ones.
Or is it django-version specific?

Reply:
`F` doesn't inherit from `BaseExpression` but from `class Combinable:`

Reply:
Oh, I am sorry :)

Reply:
No worries.

Reply:
Source: https://github.com/django/django/blob/09397f5cfade93c2e79a391ded7ab6b01e51730e/django/db/models/expressions.py#L818-L819